Dockerfile

```diff
@@ -5,6 +5,14 @@ FROM ruby:2.4.2
 RUN curl -sL https://deb.nodesource.com/setup_8.x | bash -
 RUN apt-get install -y build-essential libpq-dev nodejs
 
+# Install latest chrome dev package
+RUN set -ex; \
```

Comment:
The documentation below seems to suggest that running the tests on a headless chrome doesn't work against a docker run instance. Do we need the chrome install if that is the case?

Reply:
I'll make the documentation clearer - headless Chrome is all working fine in Docker (and necessary to run tests in Docker correctly), but not running with a visible (headful?) browser

Reply:
>headful

😂